cnf-certification-test/operator/suite.go

```diff
@@ -84,42 +82,61 @@ func testOperatorInstallationPhaseSucceeded(check *checksdb.Check, env *provider
 	check.SetResult(compliantObjects, nonCompliantObjects)
 }
 
-func testOperatorInstallationWithoutPrivileges(check *checksdb.Check, env *provider.TestEnvironment) {
+func testOperatorInstallationAccessToSCC(check *checksdb.Check, env *provider.TestEnvironment) {
 	var compliantObjects []*testhelper.ReportObject
 	var nonCompliantObjects []*testhelper.ReportObject
-	for _, op := range env.Operators {
-		check.LogInfo("Testing Operator %q", op)
-		clusterPermissions := op.Csv.Spec.InstallStrategy.StrategySpec.ClusterPermissions
+	for i := range env.Operators {
+		operator := env.Operators[i]
+		csv := operator.Csv
+		check.LogDebug("Checking operator %s", operator)
+		clusterPermissions := csv.Spec.InstallStrategy.StrategySpec.ClusterPermissions
 		if len(clusterPermissions) == 0 {
-			check.LogInfo("Operator %q has no privileged on cluster resources. No clusterPermissions found.", op)
-			compliantObjects = append(compliantObjects, testhelper.NewOperatorReportObject(op.Namespace, op.Name, "Operator has no privileges on cluster resources", true))
+			check.LogInfo("No clusterPermissions found in %s's CSV", operator)
+			compliantObjects = append(compliantObjects, testhelper.NewOperatorReportObject(operator.Namespace, operator.Name, "No RBAC rules for Security Context Constraints found in CSV", true))
```

Comment:
Add more details here to distinguish the reasons for success, for instance:
"No RBAC rules for Security Context Constraints found in CSV" -> "No RBAC rules for Security Context Constraints found in CSV (No clusterPermissions found)"

Reply:
Done in https://github.com/test-network-function/cnf-certification-test/pull/1665/commits/2d5c9de23196b0ef7a89fc01ff37cbd209dce1df